### PR TITLE
typo

### PR DESCRIPTION
**Pull request by sequencer**

### CODE REVIEW

**Comment by edwardcwang** [comment]
Note that this isn't actually a typo - by "poked", it's saying that "values [on which] `poke` [is called] get immediately passed". Probably it would be best just to get rid of that confusing word and write the sentence out fully.

**Comment by chick** [comment]
I'd either leave it the same or re-work the sentence to not use a method name as verb.

**Comment by edwardcwang** [comment]
It's not clear to me what you mean by " the clock which should be ticked by poke " in your wording.

I think what the line is basically trying to say is that outputs which are combinational paths are immediately updated without requiring a `step` after updating one of its source values (`poke`).

**Comment by grebe** [comment]
I think the most recent edit is confusing because values are passed through logic (it says "combinational logic is passed through logic").

How about:
"This is because calling `poke` on an input immediately propagates the updated values through combinational logic. Calling `step()` is only needed to update state elements in sequential logic."

**Comment by sequencer** [comment]
Sorry for my forgetfulness, I do think grebe's expression is much more clear than mine. LOL

File: 2.4_sequential_logic.ipynb [hunk 1]
```diff
@@ -111,7 +111,7 @@
     "\n",
     "Calling `step(n)` will tick the clock `n` times.\n",
     "\n",
-    "The astute observer will notice that previous testers testing combinational logic did not call `step()`. This means that the combinational logic will be immediately passed through the logic with out using 'poke' to tick the clock. \n",
+    "The astute observer will notice that previous testers testing combinational logic did not call `step()`. This is because calling poke on an input immediately propagates the updated values through combinational logic. Calling step() is only needed to update state elements in sequential logic. \n",
```
Comment (edwardcwang) [inline 2.4_sequential_logic.ipynb]: Make sure to use backticks around `poke` and `step()`

**Pull request merged**